easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -505,14 +493,14 @@ def to_str(x):
                 for key2, [def_val, _, _] in DEFAULT_CONFIG.items():
                     # only print parameters that are different from the default value
                     if key1 == key2 and val != def_val:
-                        ebtxt.append("%s = %s" % (key1, to_str(val)))
+                        ebtxt.append("%s = %s" % (key1, quote_str(val, True)))
```

Comment:
use `force_str=True` (don't specify optional arguments unnamed)

but, the whole `force_str` thing can be dropped anyway (yes, I know I suggested it ;-)), since the return value is only used in a string interpolation context anyway (and it's already part of a tuple, so no need to force it into a string anymore)